src/main/java/com/wix/mysql/config/DownloadConfigBuilder.java

```diff
@@ -4,20 +4,31 @@
 import de.flapdoodle.embed.process.config.store.DownloadPath;
 import de.flapdoodle.embed.process.config.store.TimeoutConfigBuilder;
 import de.flapdoodle.embed.process.extract.UUIDTempNaming;
+import de.flapdoodle.embed.process.io.directories.FixedPath;
+import de.flapdoodle.embed.process.io.directories.IDirectory;
 import de.flapdoodle.embed.process.io.directories.UserHome;
 import de.flapdoodle.embed.process.io.progress.StandardConsoleProgressListener;
+import org.apache.commons.lang3.StringUtils;
+
+public class DownloadConfigBuilder extends
+    de.flapdoodle.embed.process.config.store.DownloadConfigBuilder {
 
-public class DownloadConfigBuilder extends de.flapdoodle.embed.process.config.store.DownloadConfigBuilder {
 
     public DownloadConfigBuilder defaults() {
+        String artifactStoreProp = System.getProperty("mysql.artifact.store");
```

Comment:
@ezraroi while this contribution may "get the job done" it's inconsistent with the current way we define the library settings. we need to properly expose this property to allow this property to be configured.
I suggest to create a new MysqlDownloadConfig that will expose the path for developers and home dir would be the default one.